agogosml/agogosml/common/kafka_streaming_client.py

```diff
@@ -53,9 +50,26 @@ def create_kafka_config(user_config: dict) -> dict:
         config = {
             "bootstrap.servers": user_config.get("KAFKA_ADDRESS"),
             "enable.auto.commit": False,
-            "auto.offset.reset": "earliest"
+            "auto.offset.reset": "earliest",
+            # 'request.timeout.ms': 60000,
+            # 'session.timeout.ms': 60000,
```

Comment:
Should this be set to `self.timeout` defined above?

Reply:
Removing for now. these were taken from the azure tutorial but not sure if they are necessary (and it's a different timeout var than what is used below)